baseplate/observers/sentry.py

```diff
@@ -19,164 +21,133 @@
-        drilling down to an exception.
-    ``sentry.ignore_exceptions`` (optional)
-        Comma-delimited list of fully qualified names of exception classes
-        (potentially with * globs) to not report.
     ``sentry.sample_rate`` (optional)
         Percentage of errors to report. (e.g. "37%")
-    ``sentry.processors`` (optional)
-        Comma-delimited list of fully qualified names of processor classes
-        to apply to events before sending to Sentry.
+    ``sentry.ignore_errors`` (optional)
+        A comma-delimited list of exception names, unqualified (e.g.
+        ServerTimeout) or fully qualified (e.g.
+        baseplate.observers.timeout.ServerTimeout) to not notify sentry about.
+        Note: a minimal list of common exceptions is hard-coded in Baseplate,
+        this option only extends that list.
 
     Example usage::
 
-        error_reporter_from_config(app_config, __name__)
+        init_sentry_client_from_config(app_config)
 
     :param raw_config: The application configuration which should have
         settings for the error reporter.
-    :param module_name: ``__name__`` of the root module of the application.
 
     """
     cfg = config.parse_config(
         raw_config,
         {
             "sentry": {
                 "dsn": config.Optional(config.String, default=None),
-                "site": config.Optional(config.String, default=None),
                 "environment": config.Optional(config.String, default=None),
-                "include_paths": config.Optional(config.String, default=None),
-                "exclude_paths": config.Optional(config.String, default=None),
-                "additional_ignore_exceptions": config.Optional(
-                    config.TupleOf(config.String), default=[]
-                ),
                 "sample_rate": config.Optional(config.Percent, default=1),
-                "processors": config.Optional(
-                    config.TupleOf(config.String),
-                    default=["raven.processors.SanitizePasswordsProcessor"],
-                ),
+                "ignore_errors": config.Optional(config.TupleOf(config.String), default=()),
             }
         },
     )
 
-    application_module = sys.modules[module_name]
-    module_path = os.path.abspath(application_module.__file__)
-    directory = os.path.dirname(module_path)
-    release = None
-    while directory != "/":
-        try:
-            release = raven.fetch_git_sha(directory)
-        except raven.exceptions.InvalidGitRepository:
-            directory = os.path.dirname(directory)
-        else:
-            break
-
-    all_ignore_exceptions = list(ALWAYS_IGNORE_EXCEPTIONS)
-    if cfg.sentry.additional_ignore_exceptions:
-        all_ignore_exceptions.extend(cfg.sentry.additional_ignore_exceptions)
-
-    # pylint: disable=maybe-no-member
-    client = raven.Client(
-        dsn=cfg.sentry.dsn,
-        site=cfg.sentry.site,
-        release=release,
```

Comment:
I'm curious why we no longer set the `release`

Reply:
It's built-in now https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/client.py#L66

Reply:
Oh cool!